cli/test/kots_app_test.go

```diff
@@ -0,0 +1,158 @@
+package test
+
+import (
+	"bytes"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	"github.com/replicatedhq/replicated/cli/cmd"
+	"github.com/replicatedhq/replicated/pkg/kotsclient"
+	"github.com/replicatedhq/replicated/pkg/platformclient"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+var _ = Describe("kots apps", func() {
+	t := GinkgoT()
```

Comment:
why ginkgo and not pact?

Reply:
My interpretation is that pact would be for testing the API client code itself, while ginkgo lets me also test the logic in the cobra command handler...